sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Release History
 
+## 1.3.0-beta.1 (Unreleased)
```

Comment:
Automation failed to update changelog because of the Condition in the csproj. Need to manually update.